bin/circleci/build-frontend.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
-if [[ " $TESTPILOT_ENABLE_PONTOON_BRANCHES " =~ " $CIRCLE_BRANCH " ]]; then
-    NODE_ENV=production ENABLE_PONTOON=1 npm run static
+if [[ $CIRCLE_BRANCH == 'master' ]]; then
+    NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_CONTENT=1 npm run static
```

Comment:
Yeah, after making a similar comment on #1797, I'm convinced that these should be governed by a single `DEV` flag.